examples/autocomplete.amp.html

```diff
@@ -37,12 +40,88 @@
       color: red;
       text-decoration: underline;
     }
+    .wide {
+      width: 50vw;
+    }
+    .autocomplete-selected {
+      color: blue;
+      text-decoration: underline;
+    }
+    .profile-pic {
+        width: 30px;
+        height: 30px;
+        margin-right: 10px;
+        margin-left: -5px;
+    }
+    .close-friends {
+      display: flex;
+      align-items: center;
+      line-height: 18px;
+    }
   </style>
 </head>
 <body>
-    <h1>Autocomplete</h1>
+  <h1>Autocomplete</h1>
+  <h2>Inline</h2>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
+      <amp-autocomplete inline="@" min-characters="0" suggest-first filter="none" 
+          src="/form/mention/query" query="q" on="select:AMP.setState({inlineInputSelect: event.value})">
+        <label>remote data</label><br>
+        <textarea autoexpand class="wide" name="inlineInput" placeholder="Reply"></textarea>
+        <template type="amp-mustache" id="amp-template-plain">
+          <div data-value="{{ . }}">
+            {{ . }}
+          </div>
+        </template>
+      </amp-autocomplete>
+      <amp-autocomplete inline="+" min-characters="0" suggest-first filter="prefix" filter-value="email">
+        <label>local data</label><br>
+        <textarea autoexpand class="wide" rows="2" placeholder="Reply" name="inlineInput2"></textarea>
+        <template type="amp-mustache" id="amp-template-rich">
+          <div class="close-friends" data-value="{{ email }}">
+            <amp-img
+              class="profile-pic"
+              height="20"
+              width="20"
+              layout="responsive"
+              alt="Profile picture of AMP logo"
+              src="./img/ampicon.png"
+            ></amp-img>
+            <div class="info">
+              <div>{{ name }}</div>
+              <div class="custom">{{ email }}</div>
+            </div>
+          </div>
+        </template>
+        <script type="application/json">
+          { "items" : [
```

Comment:
API nit: Is the `items` key for consistency with the `amp-list` API or can the property have differently keyed siblings?

Reply:
The API explicitly reads the "items" key in the data source _or_ the value of the "items" attribute. The latter was supported in #24473. Does that answer your question?

Reply:
I guess I'm curious about why the API was designed to take an `items` prop rather than receiving an array directly.

Reply:
Ooh, yeah it was designed to mimic `amp-list` in that respect.